.github/workflows/build.yml

```diff
@@ -17,7 +17,32 @@
 
 name: build
 
-on: [push, pull_request]
+on:
```

Comment:
Why do we remove `[push, pull_request]`?

Reply:
> Why do we remove `[push, pull_request]`?

We have to pass the inputs to the pipline for the pipeline to run. I am not sure if we can pass inputs for "on" key when we mention the push/pull action. 

We can pass them under 'env' key though.

Reply:
It's necessary.

Reply:
done. added them and also the variables